scripts/cxg_admin.py

```diff
@@ -185,6 +185,31 @@ def create_cxg_artifacts(ctx):
                     s3_uri=s3_uri,
                 )
 
+@cli.command()
+@click.pass_context
+def migrate_schema_version(ctx):
+    """
+    Populates `schema_version` for each existing dataset. Since the schema version only exists
+    in the cxg file and we don't want to open them, we will call the cellxgene explorer endpoint
+    which contains the version. This is a one-off procedure since new datasets will have
+    the version already set.
+    """
+
+    import requests as rq
+    with db_session_manager() as session:
+        click.confirm(
```

Comment:
minor but should this be outside of the context manager?

Reply:
Honestly I just copied this from another method, but if you can't establish a connection to a database, I think the prompt would be redundant.